src/Heading.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import logo from "./images-folder/black-cyf-logo.jpeg"
+
+
+function Heading() {
+  const logoimage = logo;
```

Comment:
I don't think theres any point in making a new variable here, you can just pass it straight to the `src` prop

Reply:
Deleted and updated....Thank you.